operations/src/main/java/com/testingblaze/misclib/FileHandler.java

```diff
@@ -290,6 +294,24 @@ public void toDeleteFile(String fileNameWithExtensionAndPath) {
      */
     public final class Docs {
 
+        /**
+         * return content of docx file in string format.
+         * @param fileName
+         * @return
+         */
+        private String getContentFromDocxFile(String fileName){
+            InputStream is = getClass().getClassLoader().getResourceAsStream(fileName);
```

Comment:
How about making this line a private method of the class as it is used in many methods?

Reply:
Done